src/nrniv/classreg.cpp

```diff
@@ -1,19 +1,89 @@
 #include <../../nrnconf.h>
-// interface c++ class to oc
 
-#define OC_CLASSES "nrnclass.h"
-#define EXTERNS 1
-extern void
-#include OC_CLASSES
-    ;
-
-#undef EXTERNS
-static void (*register_classes[])() = {
-#include OC_CLASSES
-    ,
-    0};
+extern void Graph_reg(), HBox_reg(), VBox_reg(), GUIMath_reg(), PWManager_reg(), GrGlyph_reg(),
```

Comment:
FYI: I can undo this.
I think it's cleaner than the `#include` method, since the function definition and array was built by having to duplicate the names w/ `#define EXTERNS`.

If we want, I can change it to an X-macro, but I personally think this is more clear.